xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/ConfigurableClassMacros.xml

```diff
@@ -170,26 +170,33 @@
 #end
 
 #*
- * Augment the $adminMenu variable with all $nameOfThisDocument (i.e. XWiki.ConfigurableClass) xobjects found on this wiki.
+ * Utility for findCustomSectionsToConfigure to get an adminMenu.
+ * $appNames list of String name of applications
+ * $adminMenu the pre-filled content for the administration menu (macro is non-destructive on this parameter)
+ * $outputList
  *###
```

Comment:
```suggestion
#**
 * Utility for findCustomSectionsToConfigure to get an adminMenu.
 *
 * @param appNames list of String name of applications
 * @param adminMenu the pre-filled content for the administration menu (macro is non-destructive on this parameter)
 * @param outputList
*#
```

I've used this format in the past, which I find easier to read as it is closer to JavaDoc syntax. See https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/macros.vm#L236 .